sodetlib/operations/power_amplifiers.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import sodetlib as sdl
+
+sdl.set_action()
```

Comment:
This needs to be
```
@sdl.set_action()
```
to act as a decorator. Also this really only needs to be added to functions that produce archived data files.